ci/Jenkinsfile.combined

```diff
@@ -1,120 +1 @@
-    )
-    booleanParam(
-      name: 'PUBLISH',
-      description: 'Trigger publishing of build results for nightly or release.',
-      defaultValue: getPublishDefault(params.PUBLISH),
-    )
-  }
-
-  stages {
-    stage('Prep') {
-      steps { script {
-        println("Current JOB: ${env.JOB_NAME}")
-        /* just for a shorter access */
-        btype = utils.getBuildType()
-      } }
-    }
-    stage('Build') {
-      parallel {
-        stage('iOS') { steps { script {
-          ios = jenkins.Build('status-mobile/platforms/ios')
-        } } }
-        stage('Android') { steps { script {
-          apk = jenkins.Build('status-mobile/platforms/android')
-        } } }
-        stage('Tests') { steps { script {
-          jenkins.Build('status-mobile/platforms/tests')
-        } } }
-      }
-    }
-    stage('Archive') {
-      steps { script {
-        sh('rm -f pkg/*')
-        jenkins.copyArts(ios)
-        jenkins.copyArts(apk)
-        sha = "pkg/${utils.pkgFilename(ext: 'sha256')}"
-        dir('pkg') {
-          /* generate sha256 checksums for upload */
-          sh "sha256sum * | tee ../${sha}"
-          archiveArtifacts('*')
-        }
-      } }
-    }
-    stage('Upload') {
-      steps { script {
-        /* object for easier URLs handling */
-        urls = [
-          /* mobile */
-          Apk: utils.pkgUrl(apk),
-          iOS: utils.pkgUrl(ios),
-          Diawi: utils.getEnv(ios, 'DIAWI_URL'),
-          /* upload the sha256 checksums file too */
-          SHA: s3.uploadArtifact(sha),
-        ]
-        /* add URLs to the build description */
-        jenkins.setBuildDesc(urls)
-      } }
-    }
-    stage('Publish') {
-      when { expression { params.PUBLISH } }
-      steps { script {
-        switch (btype) {
-          case 'nightly':
-            /* Create JSON file with newest build URLs */
-            s3.updateBucketJSON(urls, 'latest.json');
-            build(job: 'website/status.im', wait: false);
-            break;
-          case 'release':
-            github.publishReleaseFiles(repo: 'status-mobile');
-            break;
-        }
-      } }
-    }
-  }
-}
-
-/* Helper that generates list of available choices for a parameter
- * but re-orders them based on the currently set value. First is default. */
-def List genChoices(String previousChoice, List defaultChoices) {
-  if (previousChoice == null) {
-     return defaultChoices
-  }
-  choices = defaultChoices.minus(previousChoice)
-  choices.add(0, previousChoice)
-  return choices
-}
-
-/* Helper that makes PUBLISH default to 'false' unless:
- * - The build is for a release branch
- * - A user explicitly specified a value
- * Since release builds create and re-create GitHub drafts every time. */
-def Boolean getPublishDefault(Boolean previousValue) {
-  if (env.JOB_NAME.startsWith('status-mobile/release')) { return true }
-  if (previousValue != null) { return previousValue }
-  return false
-}
+Jenkinsfile.combined
```

Comment:
What happened here?

Reply:
wowza, must be a bad rebase on my end, will revert this file!

Reply:
reverted.